src/components/DiagramFrame/DiagramFrame.tsx

```diff
@@ -138,7 +140,7 @@ export const DiagramFrame: React.FC<DiagramFrameProps> = ({
               borderRadius: "10px"}}
           />
           <IconButton
-            icon="GearOutline"
+            icon={<AccountTree />}
```

Comment:
Expect to be Settings from material-outline (I think thats the package, check with Jared)

Reply:
We using a new icon 
![image](https://user-images.githubusercontent.com/56001784/115912386-1b603580-a424-11eb-8b3b-af4bed546d79.png)